felix/policysync/processor_test.go

```diff
@@ -40,6 +40,19 @@ const ProfileName = "testpro"
 const TierName = "testtier"
 const PolicyName = "testpolicy"
 
+func withTimeout(timeout string, fn func()) func() {
+	return func() {
+		done := make(chan struct{})
+
+		go func() {
+			fn()
+			close(done)
+		}()
+
+		Eventually(done, "2s").Should(BeClosed())
```

Comment:
This will be a very cryptic failure - should at a minimum include a message to give some context ('e.g., "Function X timed out")

Should probably also be an `EventuallyWithOffset` so that the line number reported is correct.

Reply:
:+1:  now its `Eventually(done, "2s", Offset(1))` :grin: